routes/web.php

```diff
@@ -20,6 +23,11 @@
 
 Route::get('/', static fn () => Inertia::render('Index'));
 
+Route::get('/login', Login::class)->name('login');
+Route::get('/admin/login', static fn () => redirect('/login'));
+Route::get('/logout', Logout::class)->name('logout');
+Route::get('/auth/callback', Callback::class)->name('auth.callback');
```

Comment:
Are these `->name()` calls required for the routes to work correctly with Laravel Nova and/or the Laravel Auth0 SDK, or are they optional? If they're optional I'd leave them out at this point because it could be unclear later if anything actually uses them.

Reply:
They are indeed optional, so I removed them.